waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -287,3 +301,22 @@ genEnvValidationScript =
     [ C.mkTmplFd [relfile|scripts/validate-env.mjs|],
       C.mkUniversalTmplFdWithDst [relfile|validators.js|] [relfile|scripts/universal/validators.mjs|]
     ]
+
+genWebSockets :: AppSpec -> Generator [FileDraft]
+genWebSockets spec
+  | AS.WS.areWebSocketsUsed spec =
+      sequence
+        [ genFileCopy [relfile|webSocket.ts|],
+          genWebSocketProvider spec
+        ]
+  | otherwise = return []
+  where
+    genFileCopy = return . C.mkSrcTmplFd
+
+genWebSocketProvider :: AppSpec -> Generator FileDraft
+genWebSocketProvider spec = return $ C.mkTmplFdWithData tmplFile tmplData
+  where
+    maybeWebSocket = webSocket (snd (getApp spec))
```

Comment:
Nit: I'd probably use `$` here.

Reply:
... For the `getApp $ spec` part as well. Still irrelevant, you can resolve this one as you see fit.